src/lib/tableland-calls.ts

```diff
@@ -31,38 +36,49 @@ async function sendResponse(res: any) {
   return json.result;
 }
 
+// Take an Object with any symantic for key naming and return a new Object with keys that are lowerCamelCase
+// Example: `camelCaseKeys({structure_hash: "123"})` returns `{structureHash: "123"}`
+function camelCaseKeys(obj: object) {
```

Comment:
The RPC results body has the convention of snake_case, and we want the convention of camelCase.

Reply:
Is this just because JS likes camelCase?

Reply:
The way you've done this is quite clean, so I like it. I suspect this will make working on the SDK easier, but it's also possible it will trip folks up; perhaps one might read the RPC response structure, and unless one happens to know the  case is transformed, find it more difficult to map those things in when reading the SDK. Occasionally when debugging, I find a simple "CTRL + F" in the relevant codebase helps, and this means that wouldn't yield results in some cases.

Reply:
> Is this just because JS likes camelCase?

I didn't think to much on this because our current linting wants everything lowerCamel.  In general I think the Javascript community prefers object properties to be lowerCamel, myself included, but I know a lot of libraries don't do this.  I don't have a strong opinion, as long as we stay consistent.

> The way you've done this is quite clean, so I like it. I suspect this will make working on the SDK easier, but it's also possible it will trip folks up; perhaps one might read the RPC response structure, and unless one happens to know the case is transformed, find it more difficult to map those things in when reading the SDK. Occasionally when debugging, I find a simple "CTRL + F" in the relevant codebase helps, and this means that wouldn't yield results in some cases.

This is a good point.  As mentioned above, I have a slight preference for lowerCamel, but it's definitely just a preference.